src/components/InsureeSummary.js

```diff
@@ -93,6 +106,23 @@ const InsureeSummary = (props) => {
                   }
                 />
               </Box>
+              {showInsureeSummaryAddress && <Box>
+                <ControlledField
+                  module="insuree"
+                  id="InsureeSummary.insureeLocation"
+                  field={
+                    <Grid item xs={12}>
+                      <Typography className={classes.rawValue}>{
+                        formatMessageWithValues(props.intl, "insuree", "InsureeSummary.insureeLocation", 
```

Comment:
No need to use '**props.**intl' here since you've took out the intl from props before.